Resources/Prototypes/GameRules/events.yml

```diff
@@ -67,24 +67,24 @@
     duration: 1
   - type: ClericalErrorRule
 
-#- type: entity
-#  parent: BaseGameRule # DeltaV - this shit is busted most of the time
```

Comment:
Very bad; this shit is indeed busted, remember when it caused a CTD when getting gibbed?